frontend/src/app/projects/models/diagrams/model-diagram-dialog/model-diagram-dialog.component.html

```diff
@@ -13,112 +13,157 @@ <h2>View diagrams</h2>
       }}</span
     >
 
-    <mat-form-field
-      style="margin-bottom: -1.25em"
-      id="search"
-      appearance="outline"
-    >
-      <mat-label>Search</mat-label>
-      <input
-        [(ngModel)]="search"
-        autocomplete="off"
-        matInput
-        placeholder="Diagram name or uuid"
-      />
-      <mat-icon matSuffix>search</mat-icon>
-    </mat-form-field>
-    <span *ngIf="filteredDiagrams()"
-      >{{ filteredDiagrams()?.length }} diagram(s) found:
-    </span>
-  </div>
-  <div class="flex flex-wrap">
-    <mat-card
-      class="diagram-card"
-      #diagram
-      [id]="diagram.uuid"
-      [attr.success]="diagram.success"
-      *ngFor="let diagram of filteredDiagrams()"
-    >
-      <img
-        (click)="openModelDiagramPreviewDialog(diagram)"
-        matTooltip="Open preview"
-        *ngIf="diagram.success && diagrams[diagram.uuid]?.content"
-        class="diagram"
-        [src]="diagrams[diagram.uuid].content"
-      />
+    <br />
+    <div>
+      <mat-expansion-panel color="bg-slate-400">
+        <mat-expansion-panel-header>
+          <mat-panel-title>
+            Learn how to use the diagram cache with capellambse!
+          </mat-panel-title>
+        </mat-expansion-panel-header>
+        <div class="relative flex flex-col">
```

Comment:
Make sure that the upper and lower margins are the same: 
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/f702b28e-fdfc-4a95-8b00-aab235218c69)

Also add a bit more information what capellambse is about. Something like: 

"You can also access the diagrams via our Python library `capellambse`. Follow the installation instructions on [Github link] and then use the code snippet. To authenticate, you have to insert a personal access token. The token has the same scope as your user. Be careful with it."